src/assets/styles/components/playlist.scss

```diff
@@ -27,13 +27,22 @@
 
                 .cld-plw-item-info-wrap {
                     left: 150px;
-                    padding: 20px 15px;
+                    padding: 15px 10px;
                     top: 0;
+                    max-width: calc(100% - 150px);
```

Comment:
```
https://caniuse.com/#feat=calc
```

Let's make sure (with Amit) that we're ok with this.

Reply:
Any news about this? I think `calc()` is safe enough these days, and even if not supported nothing terrible should happen in this case...

Reply:
@tsi what about flex ?

Reply:
@gilnoycloudinary can do without flex/calc